### ISSUE TITLE
[Feature Request] Manual or CSV entries 

### ISSUE THREAD

**Issue by svanhoutte** [jaroslawhartman/withings-sync#45]
Hello 

Thanks for the great work. Would it be possible to allow manual entries for the creation of the fit file. That way if you don't have a Withings scale you can still use the tool. It could take entries from simple csv file ?

**Comment by stv0g** [comment]
Agreed, this would be a nice feature. I would be willing to merge it if somebody contributes a PR.
